rasa_core/train.py

```diff
@@ -189,82 +177,171 @@ def train_dialogue_model(domain_file, stories_file, output_path,
                                                  "remove_duplicates",
                                                  "debug_plots"})
 
-    training_data = agent.load_data(stories_file, **data_load_args)
+    training_data = agent.load_data(stories_file,
+                                    exclusion_percentage=exclusion_percentage,
+                                    **data_load_args)
     agent.train(training_data, **kwargs)
-    agent.persist(output_path, dump_flattened_stories)
+    agent.persist(output_path, dump_stories)
 
     return agent
 
 
 def _additional_arguments(args):
     additional = {
-        "epochs": args.epochs,
-        "batch_size": args.batch_size,
-        "validation_split": args.validation_split,
         "augmentation_factor": args.augmentation,
-        "debug_plots": args.debug_plots,
-        "nlu_threshold": args.nlu_threshold,
-        "core_threshold": args.core_threshold,
-        "fallback_action_name": args.fallback_action_name
+        "debug_plots": args.debug_plots
     }
 
     # remove None values
     return {k: v for k, v in additional.items() if v is not None}
 
 
+def train_comparison_models(story_filename,
+                            domain,
+                            output_path=None,
+                            exclusion_percentages=None,
+                            policy_configs=None,
+                            runs=None,
+                            dump_stories=False,
+                            kwargs=None):
+
+    """Trains either a KerasPolicy model or an EmbeddingPolicy, excluding a
+    certain percentage of a story file"""
+    for r in range(cmdline_args.runs):
+        logging.info("Starting run {}/{}".format(r + 1, cmdline_args.runs))
```

Comment:
is this relevant enough for the user to be `info` or should it be `debug`

Reply:
i'd say so, yes. knowing progress of training is standard info, shouldn't need the debug flag for that